### ISSUE TITLE
[webconf] Change "SD Card" to "Storage" and provide aview of the current storage in use

### ISSUE THREAD

**Issue by tithrion** [zynthian/zynthian-issue-tracking#1310]
**Is your feature request related to a problem? Please describe.**
As I set up my Zynthian v5.1 based on a NVME hat and according storage I regularly don't have a sd-card plugged in.
I found that webconf under System shows "SD Card: NA (NA/NA)"
(This is under Vangelis.)

**Describe the solution you'd like**
I'd prefer a more general description like "Storage" and show the Usage of the storage in use regardless of whether it is an sd-card, NVME or USB.

**Describe alternatives you've considered**
df -h on the CLI? ;-)

**Additional context**
![image](https://github.com/user-attachments/assets/55407521-1cad-4824-860a-aff5aeed780a)

**Comment by riban-bw** [comment]
Good catch! I noticed that recently but was busy with other stuff so, promptly forgot!

I have changed webconf in testing (vangelis) branch to now detect the root mount device and show its info.
![image](https://github.com/user-attachments/assets/a80e46e8-eb5e-4f8d-81f0-5ebd539676ef)

Fixed in testing (vangelis) ready for deploy in next point release.

**Comment by tithrion** [comment]
> Good catch! I noticed that recently but was busy with other stuff so, promptly forgot!

I feel you by heart.. :-) I think that's why there is an issue tracker :-D
THX for the quick fix.
I already thought 'bout doing it myself and send a pull request, but family and so on...

**Comment by tithrion** [comment]
Just updated and works as expected!

**Comment by riban-bw** [comment]
Leaving open so that it gets captured for migration to stable.
